### PR TITLE
Issue 1503 pp strength during import key without pp

### PR DESCRIPTION
**Pull request by michael-volynets**
Issue #1503, 
Time Spent: 7h (4h dev, ~3h CSS corrections, testing and fixing)

I replaced all `Settings.renderPwdStrength` with the new method `keyImportUi.validateInputPP` that generates all necesarry HTMLElements to show to user PP validation.

### CODE REVIEW

File: extension/chrome/settings/modules/backup.ts [hunk 1]
```diff
@@ -53,7 +55,7 @@ Catch.try(async () => {
     $('#' + name).css('display', 'block');
   };
 
-  $('#password').on('keyup', Ui.event.prevent('spree', () => Settings.renderPwdStrength('#step_1_password', '#password', '.action_password')));
+  keyImportUi.validateInputPP($('#password'), $('.action_password'), true);
```
[the remark this answers is not in the document]
Comment (michael-volynets) [reply extension/chrome/settings/modules/backup.ts]: Agree, that sounds better, will rename

File: test/source/tests/page_recipe.ts [hunk 2]
```diff
@@ -103,6 +103,10 @@ export class SetupPageRecipe extends PageRecipe {
         if (!/^[A-Z0-9]{4}-[A-Z0-9]{4}-[A-Z0-9]{4}-[A-Z0-9]{4}$/.test(generatedPp)) {
           throw new Error(`Import naked key page did not generate proper pass phrase, instead got: ${generatedPp}`);
         }
+        const ppValidationResult = await PageRecipe.getElementPropertyJson((await settingsPage.waitAny('.password_feedback')), 'textContent');
```
Comment (tomholub) [inline test/source/tests/page_recipe.ts]: you can just use `await settingsPage.read('...')`

Comment (tomholub) [reply test/source/tests/page_recipe.ts]: @michael-volynets please use only `@test-tags` in tests where possible

**Comment by tomholub** [comment]
I merged this by error, still need to fix

**Pull request merged**